src/csharp/Grpc.Tools.Tests/Grpc.Tools.Tests.csproj

```diff
@@ -7,12 +7,21 @@
 
   <Import Project="..\Grpc.Tools\SourceLink.csproj.include" />
 
+  <ItemGroup>
+    <!-- Prevent tests data files from being compiled into this project -->
+    <!-- TODO(jtattermusch): find a better solution for this. -->
+    <Compile Remove="IntegrationTests\**" />
+    <EmbeddedResource Remove="IntegrationTests\**" />
+    <None Remove="IntegrationTests\**" />
+  </ItemGroup>
+
```

Comment:
I did have the tests in a separate directory outside of the project but moved them back under the project to make it self contained. We could move them again if needed then these lines can be removed.